<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PairwiseRegistrationDialog</class>
 <widget class="QDialog" name="PairwiseRegistrationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>855</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pairwise Registration Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/Registar.qrc">
    <normaloff>:/images/bunny.png</normaloff>:/images/bunny.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="minimumSize">
      <size>
       <width>800</width>
       <height>600</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Target</string>
         </property>
         <property name="buddy">
          <cstring>targetComboBox</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Normal Angle Threshold</string>
         </property>
         <property name="buddy">
          <cstring>normalDoubleSpinBox</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Distance Threshold</string>
         </property>
         <property name="buddy">
          <cstring>distanceDoubleSpinBox</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>ICP iteration number</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QSpinBox" name="icpNumberSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="methodComboBox">
         <item>
          <property name="text">
           <string>Point to Point</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Point to Plane</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Point to MLSSurface</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="targetComboBox"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Source</string>
         </property>
         <property name="buddy">
          <cstring>sourceComboBox</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDoubleSpinBox" name="distanceDoubleSpinBox">
         <property name="decimals">
          <number>8</number>
         </property>
         <property name="maximum">
          <double>999.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.001000000000000</double>
         </property>
         <property name="value">
          <double>0.120000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="sourceComboBox"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Method</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QDoubleSpinBox" name="normalDoubleSpinBox">
         <property name="maximum">
          <double>180.000000000000000</double>
         </property>
         <property name="value">
          <double>45.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>MLS Search Radius</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QDoubleSpinBox" name="mlsRadiusDoubleSpinBox">
         <property name="decimals">
          <number>8</number>
         </property>
         <property name="maximum">
          <double>999.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.001000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="boundaryTestCheckBox">
       <property name="text">
        <string>Boundary Test</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="scalingCheckBox">
       <property name="text">
        <string>Allow Scaling</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="initializePushButton">
         <property name="text">
          <string>(Re)Initialize</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QPushButton" name="prePushButton">
         <property name="text">
          <string>Pre-Correspondences</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="icpPushButton">
         <property name="text">
          <string>start ICP</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QPushButton" name="exportPushButton">
         <property name="text">
          <string>Export Transformation</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="defaultPushButton">
         <property name="text">
          <string>Default</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="manualPushButton">
         <property name="text">
          <string>Manual Registration</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="helpPushButton">
         <property name="text">
          <string>Help</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="closePushButton">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="6" column="0">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>RMS error</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Rotation</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QTextEdit" name="RTextEdit">
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Translation</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Scale</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="cLineEdit">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QTextEdit" name="TTextEdit">
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>T</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>axis</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLineEdit" name="tLineEdit">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLineEdit" name="axisLineEdit"/>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>angle</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="angleLineEdit"/>
       </item>
       <item row="7" column="0">
        <widget class="QLineEdit" name="errorLineEdit"/>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>Correspondence Number</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="corrNumberLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>17</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/Registar.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closePushButton</sender>
   <signal>clicked()</signal>
   <receiver>PairwiseRegistrationDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>323</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
